.github/workflows/publish.yaml

```diff
@@ -5,6 +5,7 @@ name: Publish
 on:
   pull_request:
     branches: [ master ]
+    types: [opened, synchronize, reopened, labeled, unlabeled]
```

Comment:
Can you mention the reason for this change in the commit message.

Reply:
done